kf_lib_data_ingest/common/concept_schema.py

```diff
@@ -102,20 +102,32 @@ class DIAGNOSIS(PropertyMixin):
         CATEGORY = None
         UBERON_TUMOR_LOCATION_ID = None
         EVENT_AGE_DAYS = None
+        VERIFICATION = None
+
+        class ABATEMENT_EVENT_AGE(QuantityMixin):
+            pass
 
         class EVENT_AGE(QuantityMixin):
             pass
 
+        class ANATOMY_SITE:
+            NAME = None
+            NCIT_ID = None
+            UBERON_ID = None
+
+        HPO_ID = None
         MONDO_ID = None
         NCIT_ID = None
         ICD_ID = None
 
     class PHENOTYPE(PropertyMixin):
         NAME = None
         HPO_ID = None
+        MONDO_ID = None
```

Comment:
mondo is for diseases not phenotypes :|

Reply:
MONDO IS for diseases, not phenotypic features.